pkg/controllers/interface_filter.go

```diff
@@ -30,3 +30,13 @@ func getDefaultNetwork(networks []v1.Network) *v1.Network {
 func isDefaultNetwork(net v1.Network) bool {
 	return net.Pod != nil || (net.Multus != nil && net.Multus.Default)
 }
+
+func FilterInterfacesWithNoName(ifaces []v1.VirtualMachineInstanceNetworkInterface) []v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
I think filter functions need to be named according what they do keep
`FilterNamedInterfaces`

And then the logic might be something like 
```
if iface.Name == "" {
continue
}
```
if desirable - "return early"

Reply:
Changed the name.